pages/asctb-azimuth-data-comparison.html

```diff
@@ -52,8 +52,7 @@
                     <a class="dropdown-item" href="https://portal.hubmapconsortium.org/ccf-eui" title="Opens external website" target="_blank">CCF Exploration User Interface (EUI)</a>
                     <a class="dropdown-item" href="https://expand.iu.edu/browse/sice/cns/courses/hubmap-visible-human-mooc" title="Opens external website" target="_blank">Visible Human MOOC</a>
                     <a class="dropdown-item" href="https://portal.hubmapconsortium.org/" title="Opens external website" target="_blank">HuBMAP Data Portal</a>
-                    <a class="dropdown-item" href="kaggle.html" title="Opens external website" target="_blank">Kaggle #1: HuBMAP - Hacking the Kidney</a>
-                    <a class="dropdown-item" href="kaggle2.html" title="Opens external website">Kaggle #2: HuBMAP + HPA - Hacking the Human Body</a>
+                    <a class="dropdown-item" href="kaggle.html" title="Opens external website" target="_blank">Kaggle Competition and Awards</a>
```

Comment:
Revert this

Reply:
Exactly what should I revert here, Bruce?

Reply:
Remove the green line and re-add the red lines